charts/telegraf/values.yaml

```diff
@@ -180,3 +180,8 @@ pdb:
   create: true
   minAvailable: 1
   # maxUnavailable: 1
+
+containerPorts:
+  - name: http
+    containerPort: 9273
+    protocol: TCP
```

Comment:
@neilkuan thanks for the PR! I'm happy to take this I think, although I'm a little surprised no one has ever requested this before.

In either case, shouldn't this section be commented out? Otherwise a user with the default config would always have port 9273 exposed?

Reply:
Good catch. 
I will comment out it. Thanks for Review.